bigquery/load.go

```diff
@@ -83,6 +87,9 @@ func (l *LoadConfig) toBQ() (*bq.JobConfiguration, io.Reader) {
 			ProjectionFields:                   l.ProjectionFields,
 		},
 	}
+	if l.HivePartitioningOptions != nil {
+		config.Load.HivePartitioningOptions = l.HivePartitioningOptions.toBQ()
```

Comment:
`toBQ` works with `nil` receivers. So, you can inline this like all of the others.

https://github.com/googleapis/google-cloud-go/blob/bc3bf1cdad3aaedc6fdc2987d6cbf8a3481cadeb/bigquery/external.go#L463-L466

I see `nil` is checked for `HivePartitioningOptions.toBQ` in several places. Those can all be simplified (either in this PR or a followup one).

Reply:
done.